docs/dev/connectors/cassandra.md

```diff
@@ -72,10 +72,13 @@ The following configuration methods can be used:
 4. _setMapperOptions(MapperOptions options)_
     * Sets the mapper options that are used to configure the DataStax ObjectMapper.
     * Only applies when processing __POJO__ data types.
-5. _enableWriteAheadLog([CheckpointCommitter committer])_
+5. _setMaxConcurrentRequests(int maxConcurrentRequests, long timeout, TimeUnit unit)_
+    * Sets the maximum allowed number of concurrent requests with a timeout for acquiring permits to execute.
+    * Only applies when __enableWriteAheadLog()__ is not configured.
```

Comment:
Yeah. It was just implementation effort. I was thinking of a follow up ticket to implement it and verify it, since the WAL version is always the tricky one.